elements/pfe-navigation/src/pfe-navigation-item.scss

```diff
@@ -121,8 +137,10 @@ $variables: (
   background-color: #{pfe-local(BackgroundColor)};
   color:            #{pfe-local(Color)};
   
-  padding: #{pfe-local($cssvar: Padding, $region: trigger, $fallback: #{pfe-var(container-padding)})} !important;
-
+  padding: 10px;//#{pfe-local(Padding)};
+  :host(:not([pfe-icon])) & {
+    padding: 12px ;//#{pfe-local(Padding)};
+  }
```

Comment:
@chrisdo1 should we put back the pfe-local vars here, this looks like a mid-debugging thing

Reply:
fixed the trigger--Padding variables